sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/EpochTracker.scala

```diff
@@ -28,7 +28,7 @@ object EpochTracker {
   // update the underlying AtomicLong as it finishes epochs. Other code should only read the value.
   private val currentEpoch: InheritableThreadLocal[AtomicLong] = {
     new InheritableThreadLocal[AtomicLong] {
-      override protected def childValue(parent: AtomicLong): AtomicLong = {
+      override def childValue(parent: AtomicLong): AtomicLong = {
```

Comment:
```
[ERROR] [Error] /spark/sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/EpochTracker.scala:31: weaker access privileges in overriding
private[package lang] def childValue(x$1: java.util.concurrent.atomic.AtomicLong): java.util.concurrent.atomic.AtomicLong (defined in class ThreadLocal)
  override should at least be private[lang]
```

Reply:
It seems that this change is due to [https://github.com/scala/scala/pull/9542](https://github.com/scala/scala/pull/9542)